Packs/CriminalIP/Integrations/CriminalIP/CriminalIP.py

```diff
@@ -0,0 +1,271 @@
+import demistomock as demisto  # for XSOAR testing
+from CommonServerPython import *  # XSOAR common functions
+from CommonServerUserPython import *  # XSOAR user functions
+
+import datetime
+import dateparser
+from typing import Dict, Any, List
+
+BASE_URL = "https://api.criminalip.io/"
```

Comment:
please remove it from the global scope, please define it in the main function, and save it as property in the client class.